### DISCUSSION TITLE
Piechart navigation

### DISCUSSION THREAD

**Discussion by DaanKorver** [JustinLung/the-game-of-tones-chapterpage#12] [Questions] [answered]
Is het een mogelijkheid om de sidebar navigation (pie chart) te exporteren naar een SVG zodat we deze kunnen gebruiken?
![Screenshot from 2021-12-13 11-39-36](https://user-images.githubusercontent.com/30351629/145797775-7ea72498-4d16-43a7-aeb6-0a7d2b9c45fc.png)

**Comment by stellaniewenhuis** [comment, marked as answer]
Deze heb ik als het goed is in whatsapp gestuurd als svg code in lagen.
